react-components/src/components/AccountTrigger/accountTrigger.js

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+ *
+ *    Copyright 2019 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactDOM from 'react-dom';
+import { User as UserIcon } from 'react-feather';
+
+import { useUserContext } from '../../context/UserContext';
+import Icon from '../Icon';
+import AccountIconText from '../AccountIconText';
+
+import classes from './accountTrigger.css';
+
+const parentEl = document.querySelector('.header__accountTrigger');
+
+const AccountTrigger = () => {
+    const [{ isAccountDropdownOpen }, { toggleAccountDropdown }] = useUserContext();
+
+    const iconColor = 'rgb(var(--venia-text))';
+    const svgAttributes = {
+        stroke: iconColor
+    };
+
+    const button = (
+        <button
+            className={classes.root}
+            aria-label="Toggle account dropdown"
+            onClick={() => toggleAccountDropdown(!isAccountDropdownOpen)}>
+            <Icon src={UserIcon} attrs={svgAttributes} />
+            <AccountIconText />
```

Comment:
Can we merge the `AccountIconText` component here? I imagine we could have one component that would show the label (or not) based on a prop, i.e. `<AccountTrigger label="my optional text here"/>`.

Reply:
Sure, will have a look and let you know when it's done.

Reply:
Hi @dplaton, `<AccountTrigger>` will now accept a label as a prop and render it if it's not undefined